OrbitGl/TimeGraph.cpp

```diff
@@ -763,10 +763,12 @@ void TimeGraph::DrawTracks(GlCanvas* canvas, PickingMode picking_mode) {
         std::string process_name = GOrbitApp->GetCaptureData().process_name();
         thread_track->SetName(process_name);
         thread_track->SetLabel(process_name + " (all threads)");
+        thread_track->SetNumberOfPrioritizedTrailingCharacters(strlen("(all threads)"));
```

Comment:
Can you add a string_view constant for the string " (all threads)"?

Reply:
Yes, what do you think now?

Reply:
Great. Thanks!